src/main/java/net/minecraftforge/items/pickup/ItemPickupReasons.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.items.pickup;
+
+import static net.minecraftforge.items.pickup.ItemPickupReason.getOrCreate;
+
+/**
+ * Class containing all standard ItemPickupReasons provided by Forge.
+ *
+ * @see ItemPickupReasons
+ */
+public final class ItemPickupReasons
```

Comment:
Utility classes like these should have a private constructor to prevent instantiation.